backend/src/main/java/com/redhat/cloud/notifications/db/EndpointResources.java

```diff
@@ -142,22 +118,11 @@
         });
     }
 
-    public Uni<Long> getEndpointsCountPerType(String tenant, Set<EndpointType> type, Boolean activeOnly) {
+    public Uni<Long> getEndpointsCountPerType(String accountId, Set<EndpointType> type, Boolean activeOnly) {
```

Comment:
We don't really use this from the UI (`accountId = null`). I don't think there is any harm in sharing the default endpoints (they are emails).

IIRC, only the list endpoints use this API and it passes the account's id

Reply:
I asked that because if `accountId == null` then this PR introduces a SQL query change with the `count` query. That change may not be a bad thing though.